src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx

```diff
@@ -29,7 +35,10 @@ const JourneyPatternStopPointMap = memo(
     addStopPoint,
     deleteStopPoint,
     transportMode,
+    focusedQuayId,
+    updateFocusedQuayIdCallback,
   }: JourneyPatternStopPointMapProps) => {
```

Comment:
There's almost 200 lines of business logic before we get to the rendering part here. Can you consolidate some of this aggregate hooks maybe, and separate out into a dedicated file?

Reply:
It also makes it hard to follow the changes.

Reply:
Yup, I don't like this too. There was only little amount of stuff possible to move into hooks file, so the rest got extracted into a new MarkersWrapper component. Now there will be two components of about 150 lines instead of one gigantic one. Not sure how much it makes things easier to follow in practice, but at least it's not all cramped in one file 😅